### ISSUE TITLE
Plugin forum. How to edit messages?

### ISSUE THREAD

**Issue by Evgura** [e107inc/e107#3472]
How to edit forum posts if the topic is closed?
In the web interface and in the admin panel there is no such possibility. Only through the database?

![322233](https://user-images.githubusercontent.com/25720089/46199549-c8d04280-c328-11e8-82e5-a89e0673f124.jpg)

![111222](https://user-images.githubusercontent.com/25720089/46199638-fddc9500-c328-11e8-8513-6b419dd26742.jpg)

**Comment by Moc** [comment]
I think that's intended. If a forum topic is closed it is locked and no changes are allowed. You can reopen, edit and close again.

**Comment by Evgura** [comment]
This is wrong.
To open topic you need
1.go to the root of the forum
2.to find a topic,
3.discover
4.find in this topic message
5.fix
6.then go in the root of the forum
7.to find a topic
8.shut.
In the admin panel you need to leave the ability to edit messages. There you can not edit any messages, neither open nor closed.

**Comment by Jimmi08** [comment]
He is right. I always need to unlock forum and lock it back after editing. Totally annoying.  Locking functionality should stop new posts be added.

**Comment by rica-carv** [comment]
But i think that functionality should be handled only by main admin, via admin interface....
Forum moderators should not be able to modify locked topics....

I'm user of a forum, where in the past forum moderators chnaged thinks in locked topics, and it was a huge havoc....

**Comment by Evgura** [comment]
@rica-carv So after all, the main administrator can not edit messages from the admin panel.

**Comment by rica-carv** [comment]
@Evgura Indeed, as i've stated before, main admins should be allowed to edit them...
There's a lot of main admins that can't have access to the hosting core database control panel to edit that manually...

**Comment by Vahpurr** [comment]
I wanted to add, in case anyone else is having a similar issue = we  upgraded our out of date server and could  no longer edit any threads on our forum.  The problem is in the THEME. We have not yet identified the code error in the old theme; however, when using one of the default themes that loads with a new install of e107, we can edit threads without issue.

**Comment by Jimmi08** [comment]
> I wanted to add, in case anyone else is having a similar issue = we upgraded our out of date server and could no longer edit any threads on our forum. The problem is in the THEME. We have not yet identified the code error in the old theme; however, when using one of the default themes that loads with a new install of e107, we can edit threads without issue.

1. create your own issue :)
2. check the console, look for javascript conflict (F12, second tab Console) 
3. if this happens after only upgrade the server (not changing theme or core), check error log, the theme probably uses php5 only code.  You will see error there.
